### PR TITLE
SPCTR-2201 Typography-letter-spacing for number and heading of counter block

### PR DESCRIPTION
**Pull request by rajmohanreddy774**
### Description
Added letter-spacing to Typography of number and heading of counter block

### Screenshots
https://d.pr/v/p5xyQa

### Types of changes
New feature that adds letter-spacing settings to both number heading of counter block

### How has this been tested?

- Go to styles in editor and in number typography will get to see letter spacing give value and see the changes.
- Repeat the same for heading typography.
- switch between tablet, mobile and desktop and give values changes will be reflected in frontend.

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Review by aapatre — CHANGES_REQUESTED** [review]
All good, just ensure the following:

1. Fix PHPCS issues (you can try running `phpcbf .` )
2. Do a `git pull origin next-release` on this branch and resolve conflicts (you might need to rebuild the files and commit)

**Comment by rajmohanreddy774** [comment]
> All good, just ensure the following:
> 
> 1. Fix PHPCS issues (you can try running `phpcbf .` )
> 2. Do a `git pull origin next-release` on this branch and resolve conflicts (you might need to rebuild the files and commit)

done with changes

**Review by aapatre — APPROVED** [review]
Good to go! 🚀

**Review by AbhijitNage123 — APPROVED** [review]

**Review by bsf-zanev — APPROVED** [review]
Looks good to me.

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```
Comment (bsf-zanev) [inline .vscode/launch.json:4]: This comment should not be here.

Comment (rajmohanreddy774) [reply .vscode/launch.json]: removed

**Review by RajkiranBagal — APPROVED** [review]

**Review by RajkiranBagal — APPROVED** [review]

**Pull request merged**